diff -urN libgnomecanvas-2.6.0.orig/configure.in libgnomecanvas-2.6.0/configure.in
--- libgnomecanvas-2.6.0.orig/configure.in	2004-03-22 10:02:41.047924784 +0100
+++ libgnomecanvas-2.6.0/configure.in	2004-03-22 10:02:56.800530024 +0100
@@ -46,7 +46,7 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE",[Gettext package])
 
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB eo es et fa fi fr ga he hi hr hu id is it ja kn ko li lt lv mk ml mn ms ne nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta tr uk vi wa yi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_CA en_GB eo es et fa fi fr ga he hi hr hu id is it ja kn ko li lt lv mk ml mn ms nb ne nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@ije sr@Latn sv ta tr uk vi wa yi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 gnomelocaledir='${prefix}/${DATADIRNAME}/locale'
 AC_SUBST(gnomelocaledir)
